src/openforms/forms/management/commands/check_duplicate_component_keys.py

```diff
@@ -0,0 +1,56 @@
+from typing import List
+
+from django.core.management import BaseCommand
+
+from tabulate import tabulate
```

Comment:
I don't think this dependency exists in the `stable/1.0.x` branch, so we'll have to remember to install that before running the script.